website/showcase.json

```diff
@@ -165,5 +165,12 @@
       "Blog post": "https://medium.com/vis-gl/rendering-a-minecraft-world-with-deck-gl-927b52c5c6db"
     },
     "description": "Rendering A Minecraft World With deck.gl."
+  },
+  {
+    "name": "I3S Explorer",
+    "url": "https://production.d3nof3l3x2sso4.amplifyapp.com/",
```

Comment:
As the website isn't updated that often it would be better to create a better URL, this looks like a development URL.